Description: Patch to let munin-run print its usage if no plugin is given. This
 has been fixed upstream in trunk after a rewrite, but not in tags/1.3.4
Origin: other, http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=416478
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=416478
Index: munin-1.2.6/node/munin-run.in
===================================================================
--- munin-1.2.6.orig/node/munin-run.in	2008-08-13 01:16:17.000000000 +0200
+++ munin-1.2.6/node/munin-run.in	2008-08-13 01:17:21.000000000 +0200
@@ -97,9 +97,9 @@
              "paranoia!"    => \$paranoia,
              "help"         => \$do_usage );
 
-if ($do_usage)
+if ($do_usage or ! defined($ARGV[0]))
 {
-    print "Usage: $0 [options]
+    print "Usage: $0 [options] <plugin>
 
 Options:
     --help              View this message.
